snap/snapcraft.yaml

```diff
@@ -27,59 +21,22 @@ apps:
     common-id: org.gnome.Fractal
     desktop: usr/share/applications/org.gnome.Fractal.desktop
     plugs:
-      - home
-      - removable-media
       - network
+      - network-bind
+      - network-status
       - audio-playback
 
+
 parts:
-  ffmpeg:
-    # WORKAROUND:
-    # Build from source because the ffmpeg package installs libraries as dependencies
-    # that conflict with the Gnome extension
-    plugin: autotools
-    source: https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz
-    source-checksum: sha256/57be87c22d9b49c112b6d24bc67d42508660e6b718b3db89c44e47e289137082
-    autotools-configure-parameters:
-      # WORKAROUND: Install to /usr instead of /usr/local because it's not in search paths
-      - --prefix=/usr
-      - --disable-debug
-      - --disable-doc
-      - --disable-static
-      - --enable-gpl
-      - --enable-shared
-      - --disable-ffplay
-      - --disable-devices
-      - --enable-gnutls
-      - --enable-libmp3lame
-      - --enable-libvorbis
-    build-packages:
-      - nasm
-      - libgnutls28-dev
-      - libmp3lame-dev
-      - libvorbis-dev
-    stage-packages:
-      - libmp3lame0
-    stage:
-      - -usr/include
-  gtksourceview:
-    source: https://gitlab.gnome.org/GNOME/gtksourceview.git
-    source-branch: 'gtksourceview-4-6'
-    source-depth: 1
+  libshumate:
+    source: https://gitlab.gnome.org/GNOME/libshumate.git
+    source-tag: '1.0.3'
     plugin: meson
-    meson-parameters:
-     - --prefix=/usr
-     - -Ddebug=true
-     - -Doptimization=3
-    build-packages:
-      - gettext
-      - libxml2-dev
-    override-pull: |
-      set -eux
-      snapcraftctl pull
-      sed -i 's#Werror=missing-include-dirs#Wmissing-include-dirs#g' meson.build
+    meson-parameters: [ --prefix=/usr, -Dgir=false, -Dvapi=false, -Dgtk_doc=false]
     prime:
-      - usr/lib/*/libgtksourceview*
+      - -usr/lib/*/pkgconfig
```

Comment:
We shouldn't stage pkgconfig files

Reply:
Are staging it? I think it's `- -usr/lib/*/pkgconfig`

Isn't it meant to not stage it?

Reply:
Uhh.. Got it.. You mean that instead `prime` I shoild do `stage` right?

Reply:
oh, sorry... miss-read that.  It's fine :)